src/modules/lorem/index.ts

```diff
@@ -332,7 +361,57 @@ export class LoremModule {
 
     const method = this.faker.helpers.arrayElement(methods);
 
-    return `${this[method]()}`;
+    if (typeof options === 'undefined') {
+      return `${this[method]()}`;
+    }
+
+    if (typeof options === 'number') {
+      return this.faker.lorem.text({ length: options });
+    }
+
```

Comment:
You can use https://fakerjs.dev/api/helpers.html#rangetonumber to simplify this:

```suggestion

    const length = options.length == null ? null : faker.helpers.rangeToNumber(options.length);

    if (length == null) {
      return `${this[method]()}`;
    }

```